Tests/Integration/config/messenger.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+    messenger:
+        transports:
+            async: 'in-memory://'
+        routing:
+            'Presta\SitemapBundle\Messenger\DumpSitemapMessage': async
+#
+#services:
+#    Presta\SitemapBundle\Messenger\DumpSitemapMessageHandler:
+#        arguments:
+#            $defaultTarget: '%kernel.project_dir%/public'
+#        tags: ['messenger.message_handler']
```

Comment:
I remember now why I needed to add this. The messenger configuration for testing was imported based on message bus existence, so in phpunit it was returning false, removed the condition to check how it imports